sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2669,6 +2669,25 @@ object SQLConf {
       .checkValue(_ >= 0, "The value must be non-negative.")
       .createWithDefault(8)
 
+  val NULL_AWARE_ANTI_JOIN_OPTIMIZE_ENABLED =
+    buildConf("spark.sql.nullAware.antiJoin.optimize.enabled")
+      .internal()
+      .doc("When true, NULL-aware anti join execution will be planed into " +
```

Comment:
I think this is only possible when we can buildSide data is small. Can you add to this doc?

Reply:
if it's single column NAAj, even if the buildSide is bigger than the autoBroadcaseThreshold, I think it's better to just using the optimize, because BroadcastNestedLoopJoin will be still worst than this optimization.

Reply:
I saw SPARK-32644, so it is not always able to use this optimization, isn't?